website/docs/docs/core-versions.md

```diff
@@ -14,16 +14,19 @@ dbt Core releases follow [semantic versioning](https://semver.org/). The policie
 
 ## Version support prior to v1.0
 
-- We are no longer releasing new patches for minor versions prior to v1.0
-- As of June 30, 2022, dbt Cloud will remove support for dbt Core versions older than v1.0
+- We are no longer releasing new patches for minor versions prior to v1.0.
+- As of June 30, 2022, dbt Cloud will remove support for dbt Core versions older than v1.0. At that point, we will also remove v0.20 and v0.21 from the version dropdown on this website.
+- Pages on this website include "Changelog" notes to represent changes in functionality in versions leading up to v1.0. You can also read the migration guide for a specific version for an overview of its changes.
```

Comment:
@jtcohen6 I wasn't sure what we meant by "this website" or where these changelog notes were besides in the migration guides. I tried to clarify, but feel free to change:
```suggestion
- You can read the [specific version migration guides](/guides/migration-guide/) to understand changes to each version. Each migration guide links to "Changelog" notes where you can see functionality changes in versions leading up to v1.0.
```

Reply:
Yes we should add a landing page for sure! Eventually I will create a task to sweep through all of the pages and add them, but for now we could update the [migration guides in sidebar.js](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/sidebars.js#L69):

```js
{
      type: "category",
      label: "Migration guides",
      link: {
        type: 'generated-index',
        title: 'Migration guides',
        description: 'Learn how you can migrate to the latest version of dbt Core.',
        slug: '/docs/guides/migration-guides',
      },
          items: [
            "docs/guides/migration-guide/upgrading-to-v1.1",
            "docs/guides/migration-guide/upgrading-to-v1.0",
            "docs/guides/migration-guide/upgrading-to-0-21-0",

```

Reply:
Oh nice!! I'll give this a go